rand_distr/src/normal.rs

```diff
@@ -122,14 +122,18 @@ where F: Float, StandardNormal: Distribution<F>
 /// Error type returned from `Normal::new` and `LogNormal::new`.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub enum Error {
-    /// `std_dev < 0` or `nan`.
-    StdDevTooSmall,
+    /// The mean value is too small (log-normal samples must be positive)
+    MeanTooSmall,
+    /// The standard deviation or other dispersion parameter is negative,
+    /// infinite, or not-a-number.
+    BadVariance,
```

Comment:
This is a breaking change.

Reply:
Yup. The next release will be breaking anyway I think. Uh, I should mention in the changelog.